src/api/config/routes.rb

```diff
@@ -39,6 +39,7 @@ def self.public_or_about_path?(request)
     user:       %r{[^\/]*}
   }
 
+  # rubocop:disable Style/HashSyntax
```

Comment:
I don't like to enable and disable Rubocop inline. What about using `Exclude` when defining the cop to avoid checking this cop for the whole file? :bowtie: